### PR TITLE
Add Open graph meta tags in header

### PR DESCRIPTION
**Pull request by BenoitFuric**
Add meta properties for social cards, depends if it's for the main page ou a single episode.

* [x] I am the author of this code or the code is public domain
* [x] I release this code into the public domain

### CODE REVIEW

File: PodcastGenerator/themes/default/index.php [hunk 1]
```diff
@@ -6,7 +6,35 @@
...
+    // IF name was passed, do this instead
+    if (isset($_GET[$link])) {
+        $correctepisode = array();
+        for ($i = 0; $i < sizeof($episodes); $i++) {
+            if ($episodes[$i]["episode"]["filename"] == $_GET[$link]) {
+                $correctepisode = $episodes[$i];
+                break;
+            }
+        }
+        echo '<meta property="og:title" content="' $config ["podcast_title"] . $correctepisode["episode"]["titlePG"] . '"/>';
```
Comment (emilengler) [inline PodcastGenerator/themes/default/index.php:25]: You forgot a dot after the `'` quote

File: PodcastGenerator/themes/default/index.php [hunk 2]
```diff
@@ -6,7 +6,35 @@
...
+        echo '<meta property="og:type" content="article"/>';
+        echo '<meta property="og:url" content="' . $config["url"] . 'index.php?name=' . $correctepisode["episode"]["filename"] . '"/>';
```
Comment (emilengler) [inline PodcastGenerator/themes/default/index.php:27]: Please put a space before `'"/>'` (same for all other occurrences)

File: PodcastGenerator/themes/default/index.php [hunk 3]
```diff
@@ -6,7 +6,35 @@
...
+        echo '<meta property="og:image" content="' . $config["url"] . $config["img_dir"] . 'itunes_image.jpg"/>';
+        echo '<meta property="og:description" content="' . $config["podcast_description"] . '"/>';
+    } else {
+        echo '<meta property="og:title" content="' . $config["podcast_subtitle"] . '"/>';
```
Comment (emilengler) [inline PodcastGenerator/themes/default/index.php:31]: Why use "podcast_subtitle"? I think podcast_title should be used there

File: PodcastGenerator/themes/default/index.php (hunk 2 above)
Comment (emilengler) [reply PodcastGenerator/themes/default/index.php:27]: So it would be `. ' "/>'`

File: PodcastGenerator/themes/default/index.php (hunk 3 above)
Comment (BenoitFuric) [reply PodcastGenerator/themes/default/index.php]: You right, it's an error from my custome template. I changed to podcast_title, as it was already done for the first part.

File: PodcastGenerator/themes/default/index.php (hunk 1 above)
Comment (BenoitFuric) [reply PodcastGenerator/themes/default/index.php]: My bad. Missing dot back in place.

**Comment by BenoitFuric** [comment]
Thanks for the review and sorry for this issues. It's modified.

**Comment by emilengler** [comment]
Ok, thanks. I will merge this and do some adjustments!

**Comment by emilengler** [comment]
Thanks, your code is merged now!

**Pull request closed without merging**